test_autofit/output/non_linear/LBFGS/tag/pickles/total_iterations.pickle

```diff
@@ -0,0 +1 @@
+�K.
```

Comment:
Are pickles necessary? Might make sense to gitignore them as they change with runs adding crap to the git history

Reply:
They are for the unit test, would be hard to omit them without running a full search to create them.